app/controllers/albums_controller.rb

```diff
@@ -187,7 +187,7 @@ def update
   end  
 private
   def authenticate
```

Comment:
It looks like we are using this method everywhere. Can't we move it to application_controller?